### DISCUSSION TITLE
1st order Euler discretized equation : how is it obtained?

### DISCUSSION THREAD

**Discussion by kapilVetrivel** [wkirgsn/thermal-nn#7] [General]
The below equation, how is it derived? 
Why conductance is an exponent function?
Is it intuitive to add Thermal Conductance & Power loss directly , as per the equation?

![image](https://github.com/wkirgsn/thermal-nn/assets/167061511/546f6d0d-b986-4227-ac0b-edb9b16b9bbc)

**Comment by wkirgsn** [comment]
Hi,
the equation is derived from the standard system of ODEs described by [LPTNs](https://en.wikipedia.org/wiki/Lumped-element_model#Thermal_systems).
Since it is known that thermal conductances and power losses vary with time and operation point, they are set to be the output of MLPs in the TNN framework.

Note that not the thermal conductance but rather the thermal capacitance is modeled with an exponential function.
This is an implementation detail to accommodate more efficient training with error backprop.
The thermal capacitances for each target temperature are initialized the same way but true values differ at several magnitudes.
In order to let a gradient-descent-based iterative training converge to such large varying coefficients in few epochs, the exponential function is employed here.

**Reply by kapilVetrivel** [reply]
Thanks !

Also, simplifying the above equation results in adding Power loss & Thermal Conductances together.
Is there any specific reason behind? I intuitively attempt to balance the units on both sides.

**Reply by wkirgsn** [reply]
power losses would be added with the product of thermal conductances and temperature differences. Only that way the units will match.
